src/analysis/normed_space/basic.lean

```diff
@@ -276,6 +276,19 @@ end
 lemma continuous_nnnorm : continuous (nnnorm : α → nnreal) :=
 continuous_subtype_mk _ continuous_norm
 
+/-- If `∥y∥→∞`, then we can assume `y≠x` for any fixed `x`. -/
+lemma set_of_ne_mem_of_tendsto_norm_at_top {l : filter γ} {f : γ → α}
```

Comment:
The name is a little bit weird. As we are not in the filter name space, what about `ne_mem_filter_of_tendsto_norm_at_top`?

Reply:
Done.